src/components/Calculator.js

```diff
@@ -1,29 +1,54 @@
-import React from 'react';
+import react from 'react';
+import calculate from '../logic/calculate';
 
-class Calculator extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = { calc: ['AC', '+/-', '%', '/', 7, 8, 9, 'x', 4, 5, 6, '-', 1, 2, 3, '+'] };
+class Calculator extends react.Component {
+  constructor() {
+    super();
```

Comment:
- [ ] Please `props` need to get passed to child components of the Calculate.js component, kindly pass it 🙃 😉